src/config-schemas/openmrs-esm-patient-chart-schema.ts

```diff
@@ -3,7 +3,13 @@ import {
   coreWidgetDefinitions,
   coreDashboardDefinitions,
   coreTabbedViewDefinitions
-} from "./view-components/core-views";
+} from "../view-components/core-views";
+
+import {
+  widgetCommonConfigSchema,
+  WidgetCommonConfig,
+  ExternalPatientChartConfigs
+} from "@openmrs/esm-patient-chart-widgets";
```

Comment:
The point of the mechanism to pass config objects from `patient-chart` to widgets is to avoid any dependency on `patient-chart-widgets`. Can you remove this dependency? Those things should live here in `patient-chart`.

Reply:
I have moved them over. It's introduced separation between Types for the config, and Schema for the config. That reason is patient-chart imports patient-chart-widget, and therefore it's inevitable. They are defined in 2 different modules, but I guess we can leave with that.

Reply:
patient-chart statically imports patient-chart-widgets? Where? It shouldn't...